src/components/swimlanes/swimLanes.tsx

```diff
@@ -11,94 +11,89 @@ import {
   Link,
 } from "@nypl/design-system-react-components";
 import styles from "./Swimlanes.module.css";
-import React from "react";
 import { imageURL } from "@/utils/utils";
 import appConfig from "appConfig";
 
 const SwimLanes = ({ lanesWithNumItems }) => {
-  return (
-    <>
-      {lanesWithNumItems.map((lane, key) => (
-        <Box data-testid={lane.slug} key={lane.rank} mt="xxl">
-          <Flex alignItems="baseline">
-            <Heading id={`row-heading-${lane.slug}`} level="h2" size="heading3">
-              {lane.title}
-            </Heading>
-            <Spacer />
-            <Link
-              id={`row-see-more-${lane.slug}`}
-              type="standalone"
-              href={`${appConfig.DC_URL}/collections/lane/${lane.slug}`}
-              aria-label={`See more ${lane.title.toLowerCase()}`}
-              sx={{
-                display: { sm: "none", md: "inline" },
-                fontWeight: "500",
-                alignItems: "center",
+  return lanesWithNumItems.map((lane, key) => [
```

Comment:
This refactor is great. You shouldn't need to wrap the output in an array and can return the component directly such as

```
return lanesWithNumItms.map((lane, key) => (
  <Box ...
    ...
  </Box>
));
```

`SwimLanes` is now returning an array of arrays which React knows how to render but we can simplify it.